modules/tensor_mechanics/include/vectorpostprocessors/LineMaterialRankTwoSampler.h

```diff
@@ -30,6 +30,8 @@ class LineMaterialRankTwoSampler : public LineMaterialSamplerBase<RankTwoTensor>
    * Sets up variables for output based on the properties to be output
    * @param parameters The input parameters
    */
+  static InputParameters validParams();
```

Comment:
@cticenhour I just noticed today that the new method got put between the constructor and the description of the constructor, making the comments above this invalid.  There are probably a number of these. Would you mind checking for cases like this and fixing the ones that are affected?

Reply:
Thanks for the catch - I believe this was the result of a script that moved this declaration to a particular location in the file (referenced to the position of the constructor). I'll put in a PR soon fixing this where I find it.